### PR TITLE
Accomodate new mutant order form 

### PR DESCRIPTION
**Pull request by islean**
## Description

The mutant order form has been updated. This PR allows for the new version.

### Changed

- Mutant has order form 2184.9 instead of 2184.8

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_cg -t cg -b [THIS-BRANCH-NAME] -a
    ```

### How to test

- [ ] Do ...

### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

File: tests/fixtures/orderforms/2184.9.sarscov2.xlsx
Comment (Vince-janv) [inline tests/fixtures/orderforms/2184.9.sarscov2.xlsx:1]: I'm not sure I understand the changes in the orderform. Some drop downs don't work for me and there might be duplicated data 🤔

Comment (islean) [reply tests/fixtures/orderforms/2184.9.sarscov2.xlsx:1]: True, did not notice since the columns had no update. I will correct it.

Comment (Vince-janv) [reply tests/fixtures/orderforms/2184.9.sarscov2.xlsx:1]: Maybe we can sync with AG to get on the same page?

Comment (islean) [reply tests/fixtures/orderforms/2184.9.sarscov2.xlsx:1]: Sure, pretty sure that this particular change is purely due to data being malformed when transferring from the old orderform.

Comment (islean) [reply tests/fixtures/orderforms/2184.9.sarscov2.xlsx:1]: Ah yes, that I can see now! Thought that was also from my copying but apparently not!

**Comment by islean** [comment]
Order gets parsed correctly: 
image

**Review by Vince-janv — APPROVED** [review]
Well done, this looks great 👍

**Comment by islean** [comment]
Deployed to Hasta and cg-services-prod:

```
Logging deploy ...
Getting deployer... done.
Getting last commit message and SHA... done.
Getting version of deploy scripts... /home/isak.ohlsson
done.
Log deploy... done.
cg, version 59.23.0
θ61° [isak.ohlsson@hasta:~] [P_base] 2m57s $
```

**Pull request merged**